src/logger/logger.ts

```diff
@@ -1,5 +1,22 @@
-export function logTransaction(tx: any, name: any) {
-  console.log(`Transaction ${name}: Gas used ${tx.receipt.gasUsed}, Hash ${tx.tx}\n`);
+export function logTransaction(tx: any, name: any, printLogs: boolean = false) {
+  let output = `\n   > ${"Transaction name".padEnd(20)} ${name}\n`;
+  let logs = "";
+  const excludedKeys = ["contractAddress", "cumulativeGasUsed", "status", "transactionIndex", "type", "logsBloom"];
+  for (const [key, value] of Object.entries(tx.receipt)) {
+    if (!excludedKeys.includes(key)) {
```

Comment:
Nested `ifs` are usually bad. Let's have 

```suggestion
    if (excludedKeys.includes(key)) {
      continue;
    }
```